homework/tests/Homework.spec.js

```diff
@@ -0,0 +1,145 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+const data = [
+  'Prototype',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9'
+]
+
+
+data.forEach(version => {
+  test.describe(version + ': Add', () => {
+    test('1. Add 2 and 3 results in 5', async ({ page }) => {
```

Comment:
So, based on by previous comment - here you would use:
test(\`${i}. Add 2 and 3 results in 5\`, async ({ page }) => {

Reply:
data comes as a string in the HTML report:  ${Index}. Add 2 and 3 results in 5

Reply:
Ignore my comment for this one, it was a bad suggestion from my side and I expected different outcome 😄